benches/banking_stage.rs

```diff
@@ -186,10 +174,8 @@ fn bench_banking_stage_multi_accounts(bencher: &mut Bencher) {
     });
 }
 
-#[bench]
 fn bench_banking_stage_single_from(bencher: &mut Bencher) {
-    logger::setup();
-    let tx = 10_000_usize;
+    let tx = 1_000_usize;
```

Comment:
Not great to have to change this to be so low. Can we not adjust the number of iterations that Criterion does?

Reply:
I'll look into it. The goal of the high number of iterations is to gain statistical confidence that the benchmark improved or regressed. If we use Criterion in PRs to block regressions, then lowering iterations too far may cause false negatives.

Reply:
Yea, true. I wonder what they are using for timing.